ui/package.json

```diff
@@ -106,6 +106,7 @@
     "@neutrinojs/react": "9.0.0-rc.0",
     "babel-loader": "^8.0.5",
     "chai": "^4.2.0",
+    "cross-env": "^6.0.3",
```

Comment:
We have to run `yarn add -D cross-env` to add the package instead of manually writing it. This will make sure to update `ui/yarn.lock`.